src/Aspire.Hosting.Yarp/NodeStaticBuildOptions.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace Aspire.Hosting.Yarp;
+
+/// <summary>
+/// Configuration options for Node.js static build in YARP npm resources.
+/// </summary>
+[Experimental("ASPIREHOSTING001", UrlFormat = "https://aka.ms/dotnet/aspire/diagnostics#{0}")]
+public sealed class NodeStaticBuildOptions
```

Comment:
This can be internal, we don't need to expose this when we have extension methods.